stdlib/public/core/Span/RawSpan.swift

```diff
@@ -301,11 +301,11 @@ extension RawSpan {
   @lifetime(borrow pointer)
   public init<T: BitwiseCopyable>(
     _unsafeStart pointer: UnsafePointer<T>,
-    count: Int
+    byteCount: Int
```

Comment:
This really is `count`. It is the number of `T` elements. Below, we passed `count * MemoryLayout<T>.stride` to the `byteCount`-taking initializer. It would also be source-breaking, which we will only want to do when we officially review the `RawSpan` initializers and deprecate this spelling.

For now, let's remove the changes from lines 304-308. Thanks!